docs/release-notes/2.34.0.md

```diff
@@ -0,0 +1,73 @@
+
+# Percona Monitoring and Management 2.34.0
+
+
+| **Release date:** | [January 24, 2022]                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+
+
+## Release Highlights
+
+### Various UX enhancements to Backup Management
+
+With this release, we are updating the Backup and Restore functionality with key usability enhancements, including:
+
+- Filtering on services at backup creation time. This makes it considerably easier to find the right service for a backup when working with multiple servers.
+- Automatic redirection to the **Restores** page after initiating a restore. From here you can immediately monitor the restore process, and see the restore history for your PMM instance.
+- Added information about the type of storage for the backup artifacts on the  **All backups** page. This helps you get a more comprehensive overview of your current backups.
+- Design and usability improvements on the **Create Backup** and **All Backups** page to increase consistency with other PMM UI controls.
+
+### Components Upgrade
+- List the components which are upgraded as part of the Release.
+
+## New Features
+  
+- [PMM-11031](https://jira.percona.com/browse/PMM-11031) - To enhance the DBaas experience, the UI has been extended to include the following:
+    - Network and Security - DBaaS users can set a source range from which they can connect to their DB clusters with the **Expose** field. You can modify the exposure level using the **Source Range** field. Additionally, the **Source Range** field allows you to choose the network from which you can connect.
+    - Configuring the database engine - Copy-paste the config string into the **Configuration** text box to configure the database engine.
+    - Configuring storage classes - You can choose from the available classes.
+
+- [PMM-11086](https://jira.percona.com/browse/PMM-11086) - Improved design for the **On Demand** and **Schedule Backup** cards on the **Create Backup** page. In addition, the **Created** column on the **Restores** tab has been replaced with **Started at** and **Ended** columns to provide more insight into the restore process.
```

Comment:
This probably needs to be "Improvements" as not something completely new, but an improvement to the existing presentation.  

@artemgavrilov ?

Reply:
Yes, for sure, it's just an improvement

Reply:
ok